### PR TITLE
SPCTR-1747 Fix: Undefined index for load-google-fonts-locally for new user

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1747

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Reset the site with the Astra theme activate.
- Refresh the admin. You will see the issue in debug log.
- Switch to our branch and try doing the same. The issue gets resolved.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -169,6 +169,7 @@ When you use the Spectra along with the free Astra theme, you get a huge library
 = 2.0.8 - MONDAY, 29th AUGUST 2022 =
 * Fix: Image - Box shadow not applying to a valid element.
 * Fix: Post Carousel/Grid/Masonry - Inaccurate font size for 'Read More' link was displayed in the editor for tablet preview (when mobile and tablet font sizes were set).
+* Fix: Undefined index for `load-google-fonts-locally` for new user when Astra theme activated.
```
Comment (RajkiranBagal) [inline readme.txt:172]: Fix: PHP notice for new users when they activate Astra theme.

File: compatibility/class-uagb-astra-compatibility.php [hunk 2]
```diff
@@ -44,13 +44,14 @@ public function __construct() {
 
 			$astra_settings = ( defined( 'ASTRA_THEME_SETTINGS' ) ) ? get_option( ASTRA_THEME_SETTINGS ) : '';
 
-			if ( is_array( $astra_settings ) && empty( $astra_settings['load-google-fonts-locally'] ) || false === $astra_settings['load-google-fonts-locally'] ) {
+			if ( is_array( $astra_settings ) && empty( $astra_settings['load-google-fonts-locally'] ) || array_key_exists( 'load-google-fonts-locally', $astra_settings ) && false === $astra_settings['load-google-fonts-locally'] ) {
```
Comment (RajkiranBagal) [inline compatibility/class-uagb-astra-compatibility.php:47]: Can we just check isset instead of array_key_exists?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (AbhijitNage123) [reply readme.txt]: Done

File: compatibility/class-uagb-astra-compatibility.php (hunk 2 above)
Comment (AbhijitNage123) [reply compatibility/class-uagb-astra-compatibility.php]: Done

**Review by RajkiranBagal — APPROVED** [review]

File: compatibility/class-uagb-astra-compatibility.php [hunk 3]
```diff
@@ -44,13 +44,14 @@ public function __construct() {
 
 			$astra_settings = ( defined( 'ASTRA_THEME_SETTINGS' ) ) ? get_option( ASTRA_THEME_SETTINGS ) : '';
 
-			if ( is_array( $astra_settings ) && empty( $astra_settings['load-google-fonts-locally'] ) || false === $astra_settings['load-google-fonts-locally'] ) {
+			if ( is_array( $astra_settings ) && empty( $astra_settings['load-google-fonts-locally'] ) || isset( $astra_settings['load-google-fonts-locally'] ) && false === $astra_settings['load-google-fonts-locally'] ) {
```
Comment (RajkiranBagal) [inline compatibility/class-uagb-astra-compatibility.php:47]: Add the whole thing after `||` in a separate bracket. It may generate unexpected results.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**